### ISSUE TITLE
appbutton_on_navbar_start_position pop-up window

### ISSUE THREAD

**Issue by Acid-Crash** [Aris-t2/CustomCSSforFx#17]
When using **appbutton_on_navbar_start_position** there is an issue with urlbar right offset on pop-up windows.
It doesn't have any offset so it looks kinda cropped.
![urlbaroffset](https://user-images.githubusercontent.com/32600318/32107446-4441092c-bb37-11e7-876c-a9dd99fe2981.png)

I believe this can be fixed by adding `toolbar #urlbar-container {margin-inline-end:  !important;}` and negative `margin-inline-end` for next element in toolbar (to compensate increased distance between elements)

URL used for testing purposes: 
http://on-line.vodafone.ua/ru/messenger
type any number then click on the button and a pop-up window will appear)

**Comment by Aris-t2** [comment]
I will add this to appbutton_on_navbar_start_position.css file on 1.1.7 update:
```css
#main-window[chromehidden="menubar toolbar directories extrachrome "] #nav-bar-customization-target {
  -moz-padding-end: 2px !important;
}
```

**Comment by Acid-Crash** [comment]
Great works perfect

**Comment by Acid-Crash** [comment]
fixed in 1.1.7
